janus_core/helpers/mlip_calculators.py

```diff
@@ -224,6 +224,17 @@ def choose_calculator(
         kwargs.setdefault("sevennet_config", None)
         calculator = SevenNetCalculator(model=model_path, device=device, **kwargs)
 
+    elif arch == "grace":
+        from tensorpotential.calculator import grace_fm
```

Comment:
Do we want to expose the generic ASE calculator, or are just the foundational models enough?

Reply:
i did the foundational model since that seemed general enough... does not seem specific like in mace... also their foundation models seems to be kind of weekly... I am happy to reconsider if I get arguments.

Reply:
Ok, happy to leave it for now.